### ISSUE TITLE
Mood processing

### ISSUE THREAD

**Issue by olem10** [hardillb/TRADFRI2MQTT#15]
Thanks for a great piece of software! I have been using the gateway for some months now controlling a pair of GU10 bulbs and a FLOALT panel. I am running Mosquitto and TRADFRI2MQTT on a raspberry pi 3. Nodered dashboard and ESP8266 microcontrollers are used to control the lights. 

A few weeks ago the mood control stopped working. I do not get any error messages. The gateway recognizes the mood (e.g. EVERYDAY), but there is no visible change in color temperature or brightness. I have also tried to define my own moods in the TRÅDFRI app with the same result.

**Comment by hardillb** [comment]
Do on/off commands still work?

**Comment by olem10** [comment]
Yes, on/off and dim still works. Setting the temperature for a single bulb
also works.

Mood for a single bulb/panel does not work either.

**Comment by hardillb** [comment]
OK, I'll try and have a look, but it will probably end up being next week.

I do know the gateways should all have been updated recently to a new Authentication system (see #14 ) and some other bits to support the new RGB Colour bulbs so something may have been tweaked.

**Comment by hardillb** [comment]
OK, I've reproduced this, not sure what's going on at the moment, I'll have to try and intercept the data from the mobile app again, but with the new Authentication scheme that may be a lot harder.
